WeScan/Scan/RectangleFeaturesFunnel.swift

```diff
@@ -159,9 +176,9 @@ final class RectangleFeaturesFunnel {
     
     /// Resets the matching score of all of the rectangles in the queue to 0
     private func resetMatchingScores() {
-        rectangles = rectangles.map { (rectangle) -> RectangleMatch in
-            rectangle.matchingScore = 0
-            return rectangle
+        guard !rectangles.isEmpty else { return }
+        for rectangle in rectangles {
+            rectangle.matchingScore = 1
```

Comment:
Why use 1 as the minimum score instead of the previous 0?

Reply:
I found a weird bug that caused the scanner to freeze and tracked it down to this function, when I added the guard statement and made the matching score be 1 the bug stopped happening 🤷‍♂️

Reply:
Maybe if we could figure out the exact bug then we can at least document why this works :)